src/components/container/ReasonCard/ReasonCard.scss

```diff
@@ -0,0 +1,65 @@
+.ReasonCard {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 16px;
+  max-width: 356px;
+  font-style: normal;
+
+  &__icon {
+    position: relative;
+    width: 48px;
+    height: 48px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  &__title {
+    color: var(--color-bunting);
+    max-height: 27px;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 133%;
+  }
+
+  &__description {
+    color: #7d7cac;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 150%;
+
+    flex: none;
+    order: 2;
+    align-self: stretch;
+    flex-grow: 0;
+  }
+}
+
+ul {
+  list-style: none;
+  padding: 0px;
+}
+
+.icon__background {
+  width: 100%;
+  height: 100%;
+
+  background-color: var(--color-selective-yellow);
+  border-radius: 16px;
+
+  position: absolute;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+}
+
+img {
+  position: absolute;
+  left: 25%;
+  right: 25%;
+  top: 25%;
+  bottom: 25%;
+}
```

Comment:
Please put these inside the .ReasonCard selector, to avoid altering global styles